sdk/python/feast/protos/feast/core/DataSource_pb2.pyi

```diff
@@ -223,16 +224,20 @@ class DataSource(google.protobuf.message.Message):
         @property
         def watermark_delay_threshold(self) -> google.protobuf.duration_pb2.Duration:
             """Watermark delay threshold for stream data"""
+        @property
+        def kafka_settings(self) -> feast.core.DataFormat_pb2.StreamFormat:
```

Comment:
The unit tests are failing with the kafka_settings being of type `feast.core.DataFormat_pb2.StreamFormat`. The from_proto function [here](https://github.com/feast-dev/feast/pull/4902/files#diff-fada7a84510bd3d3051cffc414ee51ed92eff4cdd4c2ac53a1b8b7c47286cab4R481) expects the kafka_settings type(and as a data source config ) to be the `feast.core.DataFormat_pb2.DataSource`.